src/shared/components/buttons/index.jsx

```diff
@@ -135,4 +137,10 @@ export const PrimaryButton =
 export const SecondaryButton =
   themr('SecondaryButton', secondaryButton)(GenericButton);
 
+export const PrimaryOutlineButton =
```

Comment:
No, no need to add `PrimaryOutlineButton` and `SecondaryOutlineButton`, please remove them.
The whole idea behind the theming is that we already have all types of button we might need:
`PrimaryButton`, `SecondaryButton`, `Button` (default button), `DangerButton`, `GhostButton`, and we only use them everywhere, including in out modal; and then we use the context theming, i.e. if we render our join commuity modal within a subtree wrapped into
```
<ThemeProvider
  theme={{
    PrimaryButton: differentPrimaryButtonTheme,
    SeconardyButton: differentSecondaryButtonTheme,
  }}
/>
```
then primary and secondary buttons in our modal will automatically pick-up these styles.